deflect/qt/QmlStreamerImpl.cpp

```diff
@@ -108,17 +110,21 @@ QmlStreamer::Impl::Impl( const QString& qmlFile, const std::string& streamHost,
     , _stream( nullptr )
     , _eventHandler( nullptr )
     , _qmlGestures( new QmlGestures )
+    , _touchInjector( new TouchInjector( *_quickWindow,
+                                         std::bind( &Impl::_mapToScene, this,
```

Comment:
Did you mean doing:
[this](const QPointF& normalizedPos) { return _mapToScene( normalizedPos ); }
instead of
std::bind( &Impl::_mapToScene, this, std::placeholders::_1 );
?
I find the second one cleaner because it automatically deduces the argument type and avoids a repetition (like the new QObject::connect syntax). Now also imagine the function had three parameters...